### ISSUE TITLE
having trouble re-creating npm build

### ISSUE THREAD

**Issue by bradedelman** [immerjs/immer#923]
## 🙋‍♂ Question

I'd like to build from src and recreate the same build in the npm distribution.  immer 9.0.12

I'm having difficulty.

For example, in the npm distribution build, 
dist/immer.umd.development.js
has 
/*#__PURE__*/
annotations on their own line.

When I build, they are not on their own line.

I also notice that the package.json files in the git repo for immer, have "version": "9.0.0-beta.1", regardless if I get tag for 9.0.12.

Anyway, the step "install the dependencies" is a bit vague, so wondering if there's additional info there?

Basically, in a Fresh Ubuntu container, I do:

apt update
apt -y install git
apt -y install npm
git clone https://github.com/immerjs/immer.git
cd immer
git checkout tags/v9.0.12
npm install
npm run build

and I get a successful build, but the dist files are different than the files in npm immer 9.0.12.

Help greatly appreciated.

**Comment by mweststrate** [comment]
Use yarn install, not npm install

**Comment by bradedelman** [comment]
thank you. that worked. :-)

It'd be great to update the docs to say "yarn install' instead "install dependencies", given how subtle the different outcome is.

**Comment by mweststrate** [comment]
PR welcome
